src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -172,6 +172,25 @@ public enum HttpStatusCode
         Unused = 306,
         UpgradeRequired = 426,
         UseProxy = 305,
+        Processing = 102,
```

Comment:
Please add these new enum values in alphabetical ordering with the rest of them above.

Reply:
Nit: I agree the new codes should be integrated with the rest, but would it not make more sense for them to be ordered by numeric value rather than by name?

Reply:
It turns out that in the ref *.CS file, they are ordered alphabetically.

In the implementation *.CS file, they are ordered numerically.

Reply:
Ah, ok.

Reply:
Will fix.